Source/Schema.NET/OneOrMany{T}.cs

```diff
@@ -291,26 +287,6 @@ public bool Equals(OneOrMany<T> other)
         /// <returns>
         /// A hash code for this instance, suitable for use in hashing algorithms and data structures like a hash table.
         /// </returns>
-        public override int GetHashCode()
-        {
-            if (this.HasOne)
-            {
-                return HashCode.Of(this.item);
-            }
-            else if (this.HasMany)
-            {
-                return HashCode.OfEach(this.collection);
-            }
-
-            return 0;
-        }
-
-        /// <summary>
-        /// Checks whether the generic T item is a string that is either null or contains whitespace.
-        /// </summary>
-        /// <returns>
-        /// Returns true if the supplied item is a string that is null or contains whitespace.
-        /// </returns>
-        private static bool IsStringNullOrWhiteSpace(T item) => item.GetType() == typeof(string) && string.IsNullOrWhiteSpace(item as string);
+        public override int GetHashCode() => this.collection is null ? 0 : HashCode.OfEach(this.collection);

```

Comment:
This can be simplified to `HashCode.OfEach(this.collection)`. `OfEach` handles null collections.